rasa/utils/tensorflow/models.py

```diff
@@ -539,6 +548,84 @@ def __init__(
         # set up tf layers
         self._tf_layers: Dict[Text, tf.keras.layers.Layer] = {}
 
+    def adjust_sparse_layers(
```

Comment:
I think the proposal we agreed upon on slack was to rename this function to `adjust_for_incremental_training`.

Reply:
Change the docstring accordingly. The description of why DenseForSparse layers should be changed should then go to `_update_dense_for_sparse_layers`.